metabase.version

```diff
@@ -1 +1 @@
-v0.40.1
```

Comment:
@chadwcarlson why are you downgrading the version here 🤔

Reply:
only to test

Reply:
okay, noted